src/main/java/com/minecolonies/coremod/blocks/ModBlocks.java

```diff
@@ -1,50 +1,116 @@
-    public static final Block blockHutFisherman           = new BlockHutFisherman();
-    public static final Block blockSubstitution           = new BlockSubstitution();
-    public static final Block blockSolidSubstitution      = new BlockSolidSubstitution();
-    public static final Block blockHutField               = new BlockHutField();
-    public static final Block blockHutGuardTower          = new BlockHutGuardTower();
-    public static final Block blockHutWareHouse           = new BlockHutWareHouse();
-    public static final Block blockConstructionTape       = new BlockConstructionTape();
-    public static final Block blockConstructionTapeCorner = new BlockConstructionTapeCorner();
-    public static final Block blockTimberFrame            = new BlockTimberFrame();
-    public static final Block blockRack                   = new BlockMinecoloniesRack();
-    public static final Block blockWayPoint               = new BlockWaypoint();
-    public static final Block blockHutBarracks            = new BlockHutBarracks();
-    public static final Block blockHutBarracksTower       = new BlockHutBarracksTower();
-    public static final Block blockInfoPoster             = new BlockInfoPoster();
-
-
-    // Deactivated for now
-    // public static final Block blockBarrel        = new BlockBarrel();
+    public static BlockHutTownHall            blockHutTownHall;
+    public static BlockHutCitizen             blockHutCitizen;
+    public static BlockHutMiner               blockHutMiner;
+    public static BlockHutLumberjack          blockHutLumberjack;
+    public static BlockHutBaker               blockHutBaker;
+    public static BlockHutBuilder             blockHutBuilder;
+    public static BlockHutDeliveryman         blockHutDeliveryman;
+    public static BlockHutBlacksmith          blockHutBlacksmith;
+    public static BlockHutStonemason          blockHutStonemason;
+    public static BlockHutFarmer              blockHutFarmer;
+    public static BlockHutFisherman           blockHutFisherman;
+    public static BlockSubstitution           blockSubstitution;
+    public static BlockSolidSubstitution      blockSolidSubstitution;
+    public static BlockHutField               blockHutField;
+    public static BlockHutGuardTower          blockHutGuardTower;
+    public static BlockHutWareHouse           blockHutWareHouse;
+    public static BlockConstructionTape       blockConstructionTape;
+    public static BlockConstructionTapeCorner blockConstructionTapeCorner;
+    public static BlockMinecoloniesRack       blockRack;
+    public static BlockTimberFrame            blockTimberFrame;
+    public static BlockWaypoint               blockWayPoint;
+    public static BlockHutBarracks            blockHutBarracks;
+    public static BlockHutBarracksTower       blockHutBarracksTower;
+    public static BlockInfoPoster             blockInfoPoster;
+    public static BlockPaperwall              blockPaperWall;
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)